strix/data_checker.py

```diff
@@ -238,23 +220,26 @@ def get_train_valid_datasets():
             for i, data in enumerate(tqdm(dataloader)):
                 bs = dataloader.batch_size
                 if exist_mask and overlap:
-                    mask_class_num = len(data[msk_key].unique())
-                    if mask_class_num > 2:
-                        msk = one_hot(data[msk_key], mask_class_num, dim=1).type(torch.bool)
+                    mask_class_num = len(data[msk_key].unique()) - 1
```

Comment:
what if `mask_class_num` = 1, program will crash? Maybe it's better to set `msk = None` if `mask_class_num` = 1. which mean only background.

Reply:
I have added a function to check `mask_class_num` whether the mask is single-class or multi-class.